### PR TITLE
Adding benchmarks for running in golem

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

**Comment by sigurdm** [comment]
cc: @sortie

File: benchmarks/query_benchmark/lib/benchmark.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/// Returns the average time to execute [f] over [iterations] repetitions.
```
Comment (szakarias) [inline benchmarks/query_benchmark/lib/benchmark.dart]: Copyright statement? In this, and others?

File: benchmarks/query_benchmark/protos/f5.proto [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+    A171 = 1;
+  }
+  enum A172 {
+    A173 = 0;
+    A174 = 1;
+    A175 = 2;
+    A176 = 3;
+    A177 = 4;
+  }
+  optional A162      a163              = 7  [default=A163];
```
Comment (szakarias) [inline benchmarks/query_benchmark/protos/f5.proto]: weird formatting here and below

File: benchmarks/query_benchmark/protos/f5.proto [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+  repeated A118 a94 = 9;
+  repeated string a95 = 10;
+}
+message A116 {
+  repeated A119 a96 = 999;
+  extensions 1000 to max;
+}
+message A113 {
+  enum A59 {
+    A120         = 1;
```
Comment (szakarias) [inline benchmarks/query_benchmark/protos/f5.proto]: Weird formatting here and below

File: benchmarks/query_benchmark/bin/decode.dart [hunk 4]
```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:query_benchmark/generated/f0.pb.dart' as f0;
+import 'package:query_benchmark/benchmark.dart';
+import 'package:query_benchmark/readfile.dart';
+
+main() {
+  String path = const String.fromEnvironment("testfile") ?? 'testdata/500.pb';
```
Comment (szakarias) [inline benchmarks/query_benchmark/bin/decode.dart]: nit: single pling

**Pull request merged**